tests/periph_rtt/main.c

```diff
@@ -48,7 +48,7 @@ void cb(void *arg)
 
 int main(void)
 {
-    test_utils_interactive_sync();
+   //test_utils_interactive_sync();
```

Comment:
Shouldn't be accidentally merged.

Reply:
Don't worry, It was pushed on purpose. I also remove the ready for build label.
This commit will be dropped as soon as the crazy bug will be fixed. But for now, I'm still hunting.